### ISSUE TITLE
Velero 1.2 panics and crash-loops

### ISSUE THREAD

**Issue by ajchiarello** [vmware-tanzu/velero#2204]
**What steps did you take and what happened:**
I've installed velero 1.2 (using the helm chart version 2.8.1). I have configured the init container to install the aws plugin and set the backup target to a minio installation on my truenas. I have been using this setup for velero 1.1 backups successfully for several months.

The install completes, but the velero container goes into a crash-loop. The following error appears in the logs:

> panic: runtime error: invalid memory address or nil pointer dereference

**What did you expect to happen:**
Velero to start and run normally

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)

* `kubectl logs deployment/velero -n velero` https://gist.github.com/ajchiarello/a8bbcf8a894ddfc8104b78967a468bce#file-velero-log

**Environment:**

- Velero version (use `velero version`): 
Client:
	Version: v1.2.0
	Git commit: 5d008491bbf681658d3e372da1a9d3a21ca4c03c
Server:
	Version: v1.2.0

- Velero features (use `velero client config get features`): 
features: <NOT SET>
- Kubernetes version (use `kubectl version`): 1.16.3
- Kubernetes installer & version: rke 1.0
- Cloud provider or hardware configuration: vmware

**Comment by skriss** [comment]
this looks like it might be the same issue that https://github.com/vmware-tanzu/velero/pull/2200 just fixed in `master`.

As a short-term workaround, you could manually patch all of your `resticrepository` resources to ensure they have a lastMaintenanceTime. You can get a list of all of the `resticrepository` resources with:

```
velero restic repo get
```

Then, for each one, do something like:
```
kubectl patch -n velero resticrepository <RESTIC_REPO_NAME> --type merge -p '{"status":{"lastMaintenanceTime":"2020-01-01T00:00:00Z"}}'
```

**Comment by ajchiarello** [comment]
@skriss 
Thanks - that does mitigate the issue.

**Comment by skriss** [comment]
:+1: we'll likely include the code fix for this issue in a v1.2.1 release coming out shortly.

**Comment by skriss** [comment]
I'm going to close this one out since it's a dupe - keep an eye out for the release in the next week or two.
